federation/manifests/federation-apiserver-deployment.yaml

```diff
@@ -23,7 +23,6 @@ spec:
         - --etcd-servers=http://localhost:2379
         - --service-cluster-ip-range={{.FEDERATION_SERVICE_CIDR}}
         - --secure-port=443
-        - --advertise-address={{.FEDERATION_API_HOST}}
```

Comment:
Following up from the previous comments. Can we please bring this back? At least on the providers where we do have an IP address.